R/pkg/R/DataFrame.R

```diff
@@ -3642,3 +3643,74 @@ setMethod("checkpoint",
             df <- callJMethod(x@sdf, "checkpoint", as.logical(eager))
             dataFrame(df)
           })
+
+
+#' cube
+#'
+#' Create a multi-dimensional cube for the SparkDataFrame using the specified columns.
+#'
+#' If grouping expression is missing `cube` creates a single global aggregate and is equivalent to
```

Comment:
the backtick doesn't work with roxygen2 - if you want, use `\code{cube}` instead